jwst/assign_wcs/util.py

```diff
@@ -614,3 +614,19 @@ def update_s_region_keyword(model, footprint):
     else:
         model.meta.wcsinfo.s_region = s_region
         log.info("Update S_REGION to {}".format(model.meta.wcsinfo.s_region))
+
+
+def velocity_correction(velosys):
+    """
+    Compute wavelength correction to Barycentric reference frame.
+
+    Parameters
+    ----------
+    velosys : float
+        Radial velocity wrt Barycenter.
```

Comment:
Might want to add units (m/s?) here, just to make it obvious for future readers.